src/main/java/co/inventrosoft/springboottask/service/MatchService.java

```diff
@@ -0,0 +1,97 @@
+package co.inventrosoft.springboottask.service;
+
+import co.inventrosoft.springboottask.console.MatchResult;
+import co.inventrosoft.springboottask.model.Match;
+import co.inventrosoft.springboottask.model.Team;
+import co.inventrosoft.springboottask.repository.MatchRepository;
+import org.springframework.stereotype.Service;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+@Service
+public class MatchService {
+    private final TeamService teamService;
```

Comment:
It is not a good idea to have a different DB service in another DB service (in future may be cause of cyclic dependencies).

Good approach: MatchTeamService has (TeamService & MatchService).